dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/startup/InitTableAlteror.java

```diff
@@ -175,6 +175,11 @@ public void execute()
         executeSql( "UPDATE dataelementcategoryoption SET publicaccess = 'rwrw----' WHERE code = 'default'" );
         executeSql( "UPDATE dataelementcategory SET publicaccess = 'rw------' WHERE code = 'default'" );
         executeSql( "UPDATE categorycombo SET publicaccess = 'rw------' WHERE code = 'default'" );
+
+        //New enum column was added into ProgramStage. I need to fill default values and make it NOT NULL
+        executeSql( "UPDATE programstage SET validation = 'NO_VALIDATION' WHERE validcompleteonly = false" );
+        executeSql( "UPDATE programstage SET validation = 'VALIDATION_ON_COMPLETE' WHERE validcompleteonly = true" );
+        executeSql( "ALTER TABLE programstage ALTER COLUMN validation SET NOT NULL" );
```

Comment:
You can also remove the validcompleteonly column as we are no longer using after this sql.